backend/src/utils/normaliser/employees.ts

```diff
@@ -0,0 +1,31 @@
+import { IClientCoursePopulated } from 'interfaces/Ientities/IclientCourses';
+import { IUser } from 'interfaces/Ientities/Iusers';
+import {
+  IEmployeeClientCourse,
+  IEmployeeInfo,
+  TEmployeeCourse,
+} from 'interfaces/IResponse/IResponse';
+
+const mapEmployeeClientCourse = (clientCourse: IClientCoursePopulated): IEmployeeClientCourse => {
+  const { status, progress, date } = clientCourse;
+  const { title, avatar } = clientCourse.course;
+  const course: TEmployeeCourse = { title, avatar };
+  const emplooyeeClientCourse: IEmployeeClientCourse = { status, progress, date, course };
+  return emplooyeeClientCourse;
+};
+
+const mapEmployeeInfo = (employee: IUser, courses: IClientCoursePopulated[]): IEmployeeInfo => ({
```

Comment:
return { ...employee, courses: course.map(mapEmployess)}

Reply:
That won't work. Typescript won't give permission for compiling this code 😢